youtube_dl/extractor/drtv.py

```diff
@@ -85,7 +97,11 @@ def _real_extract(self, url):
             kind = asset.get('Kind')
             if kind == 'Image':
                 thumbnail = asset.get('Uri')
-            elif kind in ('VideoResource', 'AudioResource'):
+            preference = 0
+
+            sign_language = asset.get('Target') == 'SignLanguage'
```

Comment:
Ok, it's fine to separate it in order not to calculate it every time in a loop over links, but it should be located near `spoken_subtitles`.